app/Http/Controllers/RegisterController.php

```diff
@@ -57,7 +57,22 @@ public function loginUser(Request $request){
                 }
     }
 
-    public function dashboard(Accountant $accountant){
-        return view('dashboard.userListing');
+    function dashboard()
+    {
+        return view('dashboard.welcome');
+    }
+
+    public function list(){
+        $accountant_data = Accountant::all();
+        return view('dashboard.userListing', ['users' => $accountant_data]);
```

Comment:
```suggestion
        return view('dashboard.userListing', ['users' => $accountantData]);
```
Use camelCase for laravel objects or variables or here you can simply use `users` as identifiers.

Reply:
Thanks for the feedback sir, I'll update code in the latest commit.